### PR TITLE
Add opts to data-set install scripts

### LINKED ISSUE #1157: zowe data set scripts don't use options identifiers for arguments
- [ ] zowe-copy-to-JES.sh
- [ ] zowe-install-proc.sh
- [ ] zowe-install-xmem.sh

use arguments without options, which means that there is ambiguity over what is passed in. This blocks #1156 

Are we allowed to get these fixed - do they count as a breaking change?

### PR DESCRIPTION
**Pull request by stevenhorsman**
#### PR type
- [X] Bugfix

#### Relevant issues
Fixes #1157 

#### Changes proposed in this PR
Add options to called scripts

#### Is there a related doc issue or Pull Request? 
Doc issue/PR number: https://github.com/zowe/docs-site/issues/1054

Needs to merge with https://github.com/zowe/zowe-install-test/pull/174

### CODE REVIEW

**Comment by stevenhorsman** [comment]
Tested in https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-174/5/

File: scripts/utils/zowe-install-xmem.sh [hunk 1]
```diff
@@ -39,40 +53,36 @@ echo "<$SCRIPT>" | tee -a ${LOG_FILE}
...
 
 # check parms
-if [[ $# -lt 3 || $# -gt 4 ]]
+if [[ -z ${data_set_prefix} || -z ${loadlib} || -z ${parmlib} ]]
 then
-echo Expected 3 or 4 parameters, found $# | tee -a ${LOG_FILE}
-echo Parameters supplied were $@ | tee -a ${LOG_FILE}
-echo Usage:
+echo Parameters supplied were $@ >> ${LOG_FILE}
+echo "Not all the required parameters were supplied"
```
Comment (John-A-Davies) [inline scripts/utils/zowe-install-xmem.sh]: It would be friendly to tell the user which parm(s) they omitted.

Comment (stevenhorsman) [reply scripts/utils/zowe-install-xmem.sh]: Updated to output the missing parameters

**Comment by stevenhorsman** [comment]
Re-tested: https://wash.zowe.org:8443/job/zowe-install-test/view/change-requests/job/PR-174/7/

**Pull request merged**